pkgs/top-level/python-packages.nix

```diff
@@ -19269,6 +19269,8 @@ in {
     };
   };
 
+  rpl = callPackage ../tools/text/rpl {};
```

Comment:
If it is not a python package, please move it to `all-packages` instead.

Reply:
it's a python program but not a library ? still going to all-packages.nix ?

Reply:
Yes. We need to build python programs only against a single python version. pythonPackages are build against all versions.